### ISSUE TITLE
 Lime3DS, citra is a dead end

### ISSUE THREAD

**Issue by Exarkuniv** [Exarkuniv/RetroPie-Extra#193]
https://github.com/Lime3DS/Lime3DS?tab=readme-ov-file

this does build, 

citra does not work, even with the new links. 
it does not like the CPU, so not even going to try anymore

**Comment by RoseEverdeen** [comment]
[citra.sh.zip](https://github.com/user-attachments/files/16069554/citra.sh.zip)
I located this script for install citra from https://github.com/NGC602/citra_yuzu.  I've tweaked it a bit to point to the new repo and ran it on ubuntu cinnamon 22.04 and i managed to compile citra.
I also noticed on the new repo they used git clone --recursive https://github.com/PabloMK7/citra
cd citra

git submodule sync
git submodule foreach --recursive git reset --hard
git submodule update --init --recursive

I'm not very versed with Linux but thought I'd let you know in case it helps.

**Comment by Exarkuniv** [comment]
i did try that one also, and it would not work for me. which is why i looked into lime, since that one compiled for me very easyly

but i will try your script

**Comment by Exarkuniv** [comment]
so i tested the schipt, and it did install, 

i just need to get some roms and test it

**Comment by StonedEdge** [comment]
This compiled for me too. I wonder if we can get the libretro core working too?

**Comment by Exarkuniv** [comment]
I have tried that. And I got nowhere. Other the. Finding more ways it will not work. 

Which is why I'm going to try lime.

But if you want to give a shot. Go for it. I'll gladly update the script if you clget it working.

**Comment by StonedEdge** [comment]
No problem. I'm running an RK3588 and I was able to get it working by forcing the GL to be 4.3, but since my GPU only supports Open GL 3.3 at the moment, was hoping to be able to use the libretro core.

**Comment by Exarkuniv** [comment]
Yes that's the best way to go. 
For the pi5 I don't think there's a way to force the openGL without the chance to breaking something else

**Comment by Exarkuniv** [comment]
found this. might be worth looking into 
https://github.com/jonian/libretro-panda3ds

**Comment by DTEAM-1** [comment]
I'm not good to create .sh file. I see this : https://github.com/PabloMK7/citra   for batocera  . They use 
      https://github.com/PabloMK7/citra

**Comment by Exarkuniv** [comment]
Yes i know of that, and i have tried to use it, and again didn't work. might try again later but for now im going to use Lime and go from there

**Comment by DTEAM-1** [comment]
They fixed something with OpenGL on july 10 under under aarch64 . Don't know if it's helpful. 
https://github.com/Lime3DS/Lime3DS/commit/b240dac871292c9bf04e581a9bcf7021861858e7
https://github.com/Lime3DS/Lime3DS/issues/124

I tried to write a Shell script with no success. I don't have enough knowlege for now. I also tried a [flatpack ](https://github.com/flathub/org.citra_emu.citra/tree/master)with no success.
